src/App.js

```diff
@@ -1,6 +1,6 @@
 import RestaurantCard from './Pages/RestaurantCard/RestaurantCard';
 import ContactForm from './Pages/ContactForm/ContactForm';
-import { Routes, Route } from 'react-router-dom';
+import { Routes, Route, Link } from 'react-router-dom';
```

Comment:
If you plan to style your links here and later, it's recommended to use NavLink instead of Link.

Reply:
cheers mate :)